### DISCUSSION TITLE
Invalid client hash Linux in 2024.130.* and 2024.131.0

### DISCUSSION THREAD

**Discussion by j4chck** [ppy/osu#26962] [Q&A] [answered]
Solo scores submitting didn't work too.

What i  tried:
1. Restart game.
2. Running without gamemoderun.
3. Reboot PC.
4. Upgrade OS.
5. Change osu! directory.

Specific system configurations: execution right in specific directories only  (execution disabled in /tmp/** and $HOME/**).

[1706881385.database.log](https://github.com/ppy/osu/files/14141317/1706881385.database.log)
[1706882101.network.log](https://github.com/ppy/osu/files/14141318/1706882101.network.log)
[1706882101.runtime.log](https://github.com/ppy/osu/files/14141319/1706882101.runtime.log)

**Comment by bdach** [comment]
Are you using the appimage? If you aren't, please try the appimage.

**Reply by j4chck** [reply]
Yep, i only use appimage

**Comment by j4chck** [comment]
Forgot to say that execution disabled at file system level and didn't work even execution flag enabled

**Comment by j4chck** [comment, marked as answer]
Fixed by setting $HOME environment variable to place where execution allowed

**Reply by j4chck** [reply]
However why game require execution permission in $HOME?
